src/components/Common/Footer/Footer.tsx

```diff
@@ -0,0 +1,24 @@
+// Footer.tsx
+
+import "./Footer.scss";
+
+/**
+ * Component representing the footer section of the website.
+ * Contains links to various pages of the website.
+ */
+const Footer = () => {
+  return (
+    <>
+      <footer className='footer-container'>
+        <div className='footer-items'>
+          <div className="footer-item"> Contact Us </div>
```

Comment:
If the items in the footer are meant to be links, it's better to use the <a> element instead of <div>.